sentinelsat/sentinel.py

```diff
@@ -939,7 +939,40 @@ def check_bounds(geometry):
     wkt = re.sub(r'(?<!\d) ', '', wkt)
     return wkt
 
+def naturalname_to_wkt(naturalname):
+    """Maps the naturalname to rectangular bounding extents using Nominatim API and converts to Well-Known-Text.
 
+    Parameters
+    ----------
+    naturalname : str
+	the query to geocode
+        
+    Returns
+    -------
+    polygon coordinates
```

Comment:
The function does not return the polygon but a tuple of the placename and the coordinates. The preferable solution would be to mirror [`geojson_to_wkt`](https://github.com/sentinelsat/sentinelsat/blob/d9333fd92fc1fdf182bf0f55da34412586427ff3/sentinelsat/sentinel.py#L889) and just return the coordinates, so it can be a drop-in replacement.

i.e.
```
products = api.query(footprint=placename_to_wkt("florida"))
```

Reply:
Yeah my bad! I will change it.
Another issue is: where the user should be intimated of the returned placename for which the query will execute. Isn't the user's approval required before executing the query as the placename may not be what he wants or simply be erroneous.

Reply:
For this feature I would prefer simplicity over explicitly knowing the place queried. The whole idea is to provide a simple search interface for people who do not want to deal with defining their area-of-interest with a polygon. So taking the first result in ambiguous cases should be fine.

If the returned bbox for a placename is ambiguous we could include this as a caveat in the docs or docstrings and point people to testing their search on nominatim first. If someone needs to be more explicit with what area they want to search providing a polygon is still the best option.

Reply:
I agree. Queries are cheap and people will hopefully check whether the results are as they expect before downloading all the data...

Reply:
ok i have made the changes.